mappings/net/minecraft/client/render/BufferBuilder.mapping

```diff
@@ -34,28 +37,28 @@ CLASS net/minecraft/class_287 net/minecraft/client/render/BufferBuilder
 	METHOD method_1328 begin (Lnet/minecraft/class_293$class_5596;Lnet/minecraft/class_293;)V
 		ARG 1 drawMode
 		ARG 2 format
-	METHOD method_1334 popState ()Lnet/minecraft/class_287$class_5594;
+	METHOD method_1334 getSortingData ()Lnet/minecraft/class_287$class_5594;
 	METHOD method_1335 grow (I)V
 		ARG 1 size
 	METHOD method_1343 clear ()V
 	METHOD method_16005 roundBufferSize (I)I
-		ARG 0 amount
+		ARG 0 size
 	METHOD method_22625 ([FII)I
 		ARG 1 a
 		ARG 2 b
 	METHOD method_22892 grow ()V
 	METHOD method_22893 isBuilding ()Z
-	METHOD method_23477 reset ()V
+	METHOD method_23477 discard ()V
```

Comment:
This is not very accurate imo, it just forwards to clear without really discarding any object like some allocated buffer or so?

Reply:
This method resets where to write the next batch, which can cause it to overwrite (possibly unused) built buffers. This effectively frees built buffers as it's not safe to use them after calling this method. If it's a stretched definition of `discard`, I'm fine with reverting changes.